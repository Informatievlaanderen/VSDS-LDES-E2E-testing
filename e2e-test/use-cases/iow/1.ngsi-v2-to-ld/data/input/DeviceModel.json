{
    "id": "urn:ngsi-v2:cot-imec-be:devicemodel:ysi-exo2",
    "type": "DeviceModel",
    "function": {
        "type": "StructuredValue",
        "value": [
            "sensing"
        ],
        "metadata": {}
    },
    "controlledProperty": {
        "type": "StructuredValue",
        "value": [
            "temperature",
            "conductivity",
            "pH",
            "O2",
            "turbidity"
        ],
        "metadata": {}
    },
    "name": {
        "type": "String",
        "value": "EXO2 multi-parameter probe",
        "metadata": {}
    },
    "iow_tags": {
        "type": "StructuredValue",
        "value": [
            "external"
        ],
        "metadata": {}
    },
    "supportedUnits": {
        "type": "StructuredValue",
        "value": [
            "CEL",
            "D10",
            "Q30",
            "M1",
            ""
        ],
        "metadata": {}
    },
    "manufacturerName": {
        "type": "String",
        "value": "Xylem",
        "metadata": {}
    },
    "modelName": {
        "type": "String",
        "value": "EXO2 Multiparameter sensor",
        "metadata": {}
    },
    "brandName": {
        "type": "String",
        "value": "YSI",
        "metadata": {}
    },
    "category": {
        "type": "StructuredValue",
        "value": [
            "sensor"
        ],
        "metadata": {}
    }
}