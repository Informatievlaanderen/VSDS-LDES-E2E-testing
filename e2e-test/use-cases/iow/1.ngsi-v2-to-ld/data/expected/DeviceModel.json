{
    "@context": [
        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"
    ],
    "id": "urn:ngsi-v2:cot-imec-be:devicemodel:imec-iow-sensor-v0005",
    "type": "DeviceModel",
    "brandName": {
        "type": "Property",
        "value": "imec"
    },
    "category": {
        "type": "Property",
        "value": [
            "sensor"
        ]
    },
    "controlledProperty": {
        "type": "Property",
        "value": [
            "temperature",
            "conductivity",
            "pH",
            "nitrate"
        ]
    },
    "function": {
        "type": "Property",
        "value": [
            "sensing"
        ]
    },
    "iow_tags": {
        "type": "Property",
        "value": [
            "wqmd_sensor"
        ]
    },
    "manufacturerName": {
        "type": "Property",
        "value": "IMEC NL"
    },
    "maxTimeBetweenObservations": {
        "type": "Property",
        "value": 60
    },
    "modelName": {
        "type": "Property",
        "value": "iow-sensor-v0005"
    },
    "supportedUnits": {
        "type": "Property",
        "value": [
            "CEL",
            "D10",
            "Q30"
        ]
    }
}